content/companies/06-gojek-tech.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Gojek Tech
+excerpt: Know more about Gojek Tech's interview process ➡️
+index: 6
```

Comment:
you can remove this `index` key. This was old design now I'm taking the filename as the index name so less hassle on the developer's end